### PR TITLE
first readme version

### PR DESCRIPTION
**Pull request by mrWojackETH**
Reademe created

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+> **Note**
+>
+> This workspace is fully customized for being used within VSC, some helper
+> extensions and bash terminal. It's recommended to follow the next steps:
+>
+> - Install following extensions: `formulahendry.auto-rename-tag`, `pranaygp.vscode-css-peek`,
+>   `usernamehw.errorlens`, `dbaeumer.vscode-eslint`, `eamodio.gitlens`, `kisstkondoros.vscode-gutter-preview`,
+>   `nicolasparada.innerhtml`, `VisualStudioExptTeam.vscodeintellicode`, `Zignd.html-css-class-completion`,
+>   `esbenp.prettier-vscode`, `TabNine.tabnine-vscode`, `bradlc.vscode-tailwindcss`,
+>   `Gruntfuggly.todo-tree`, `ChakrounAnas.turbo-console-log`.
```
Comment (venturars) [inline README.md:12]: This 2 plugins are not necessary now.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+| File(s)              | Description                                            |
+| -------------------- | ------------------------------------------------------ |
+| `next.config.js`     | Config file for Nextjs                                 |
+| `tailwind.config.js` | Config file for Tailwind and Daisy UI (design system)  |
+| `styles`             | All custom CSS are inside here                         |
+| `types`              | All custom types are inside here                       |
+| `postcss.config.js`  | Config file for Tailwind and Daisy UI                  |
+| `fonts.config.js`    | Config file for @next/fonts                            |
+| `locales.config.js`  | Custom config file for internationalization system     |
```
Comment (venturars) [inline README.md:81]: This file is deprecated

File: README.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+### Run locally in development mode
+
+```bash
+pnpm install && pnpm run dev
+```
+
+When you run this development server, the changes you make in your frontend will be applied live using hot reloading.
+
+### Deploy to production
```
Comment (venturars) [inline README.md:54]: Please check landing repo for do this section

File: README.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+| `postcss.config.js`  | Config file for Tailwind and Daisy UI                  |
+| `fonts.config.js`    | Config file for @next/fonts                            |
+| `locales.config.js`  | Custom config file for internationalization system     |
+| `.prettierrc.json`   | Config file for Prettier vsc extension                 |
+| `.eslintrc.json`     | Config file for lint                                   |
+| `.vscode`            | Specific vscode config for project                     |
+| `.npmrc`             | Config file for pnpm                                   |
+| `lefthook.yml`       | Config file for lefthook (githooks tools)              |
+| `.lefthook`          | Folder scripts for lefthook (githooks tools)           |
+| `renovate.json`      | Config file for renovate bot (Github app)              |
```
Comment (venturars) [inline README.md:88]: please put "github app" instead of 'Github app'

File: README.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+## Tools
```
Comment (venturars) [inline README.md:90]: Check lading repo for do this section

**Comment by mrWojackETH** [comment]
First readme version adapted to the other readme's style

**Review by venturars — APPROVED** [review]
Good =)

**Pull request merged**